### PR TITLE
[wgsl] bool equality

### PR DESCRIPTION
**Pull request by dneto0**
And change !== to !=

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -2143,6 +2151,16 @@ Issue: (dneto): Bitwise-complement is under discussion.  https://github.com/gpuw
   <thead>
     <tr><td>Precondition<td>Conclusion<td>Notes
   </thead>
+  <tr><td>*e1* : *T*<br>
+          *e2* : *T*<br>
+          *T* is vec*N*&lt;bool&gt;
+       <td class="nowrap">`e1 == e2` : vec*N*&lt;bool&gt;
+       <td>Component-wise equality (OpLogicalEqual)
```
Comment (dj2) [inline wgsl/index.bs:2158]: Maybe add a `e1[i] == e2[i]`?

File: wgsl/index.bs [hunk 2]
```diff
@@ -2143,6 +2151,16 @@ Issue: (dneto): Bitwise-complement is under discussion.  https://github.com/gpuw
...
+  <tr><td>*e1* : *T*<br>
+          *e2* : *T*<br>
+          *T* is vec*N*&lt;bool&gt;
+       <td class="nowrap">`e1 != e2` : vec*N*&lt;bool&gt;
+       <td>Component-wise inequality (OpLogicalNotEqual)
```
Comment (dj2) [inline wgsl/index.bs:2163]: Ditto `e1[i] != e2[i]` ?

**Comment by grorg** [comment]
Accepted at the [2020-07-14 meeting](https://docs.google.com/document/d/1vBgQ8MDch3-G5LUhrKAE3K2KYeuD5NjGnugCGv0tnVg/).

**Comment by dneto0** [comment]
I'll patch to spell out component-wise semantics, like I did for #914

**Pull request merged**